### PR TITLE
Fix CI publish step

### PR DESCRIPTION
**Pull request by anotherminh**
## Related Issues

- _[none]_

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -91,7 +91,7 @@ jobs:
           extra_args: --source ${{ github.event.pull_request.base.sha || 'HEAD~1' }} --origin ${{ github.event.pull_request.head.sha || 'HEAD' }}
 
   build-to-npm:
-    if: github.ref == 'refs/heads/main'
+    # if: github.ref == 'refs/heads/main'
```
Comment (MadDataScience) [inline .github/workflows/ci.yml:94]: Should these comments be removed? (Also L135)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -108,11 +108,31 @@ jobs:
...
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        name: Install yarn deps
+        shell: bash
+
       - name: Publish to yarn/npm
         run: yarn npm publish
 
   build-to-github-packages:
-    if: github.ref == 'refs/heads/main'
+    # if: github.ref == 'refs/heads/main'
```
Comment (dmattia) [inline .github/workflows/ci.yml:135]: nit: undo this in both places before merging please

**Review by dmattia — APPROVED** [review]

**Review by MadDataScience — APPROVED** [review]
Would love to understand what's behind this...but not now.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (anotherminh) [reply .github/workflows/ci.yml:94]: @MadDataScience ya i just wanted to test the jobs on the PR rather than on merge, will remove!

**Review by MadDataScience — APPROVED** [review]

**Pull request merged**